### PR TITLE
[NEW UI] Update page - Error handling

### PR DESCRIPTION
**Pull request by ga-devfront**
| Questions         | Answers
| ------------------| -------------------------------------------------------
| Description?      | Add error handling process (alert and buttons).
| Type?             | new feature
| BC breaks?        | no
| Deprecations?     |  no
| Fixed ticket?     | N/A
| Sponsor company   | @PrestaShopCorp
| How to test?      | Try to do an update you are sure it finish on error. WARNING: Send error report does not run for moment and restore buttons can't redirect cause page didn't exist for moment.

### CODE REVIEW

File: _dev/src/ts/components/ProgressBar.ts
Comment (Quetzacoalt91) [inline _dev/src/ts/components/ProgressBar.ts]: Why not reusing the interface we have: `Destroyable`?

**Review by Quetzacoalt91 — APPROVED** [review]

**Comment by ps-jarvis** [comment]
PR merged, well done!

Message to @PrestaShop/committers: do not forget to milestone it before the merge.

**Pull request merged**
